azurerm/resource_arm_traffic_manager_endpoint.go

```diff
@@ -110,6 +112,58 @@ func resourceArmTrafficManagerEndpoint() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"string_header": {
```

Comment:
This is a bit weird to have both this and custom_header. Could we remove this and keep the schema inline with the API?

Reply:
It is designed for easier input of a large a mount of IPs when transporting data from other platform to the Azure.
I will remove it as requested.